pennylane_lightning/core/src/simulators/lightning_qubit/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -1960,6 +1962,43 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         }
     }
 
+    template <class PrecisionT, class ParamT>
+    static void applyPSWAP(std::complex<PrecisionT> *arr,
+                           std::size_t num_qubits,
+                           const std::vector<std::size_t> &wires, bool inverse,
+                           ParamT angle) {
+        applyNCPSWAP(arr, num_qubits, {}, {}, wires, inverse, angle);
+    }
```

Comment:
Not a blocker, but if/when you have a chance to improve the documentation, having time, please do it.